sidebars/faq.js

```diff
@@ -0,0 +1,49 @@
+const sidebar = [
```

Comment:
You can leverage the pattern used in other guides to create a back button here, which will both serve as an indication that you're in an FAQ, and provide a link back to the index.